scripts/publish.sh

```diff
@@ -499,6 +525,8 @@ fi
 if [ "${1}" = "cicd-publish" ]; then
 	if [ "${2}" = "dockerhub" ]; then
 		publish_to_docker_hub amazon/aws-for-fluent-bit
+	elif [ "${2}" = "public-ecr" ]; then
+		publish_to_public_ecr amazon/aws-for-fluent-bit
```

Comment:
nit: is the argument `amazon/aws-for-fluent-bit` to the function actually needed? I think it isn't